lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -18,6 +18,11 @@ table.ccx-schedule {
     th, td {
```

Comment:
Also, how, when, and where does this Sass partial get compiled? I don't see it referenced anywhere. It'd be good to know how it's included.